bedrock/firefox/templates/firefox/features/pdf-complete-fr.html

```diff
@@ -85,10 +84,9 @@ <h2>Enregistrer sa déclaration au format PDF</h2>
 
 <p>Enregistrez tous vos PDF dans des dossiers bien nommés pour une gestion facile et sécurisée.</p>
 
-<hr>
-
-<h2>Vous n'avez pas encore Firefox ?</h2>
-<p><a href="{{ url('firefox.features.index') }}">En savoir plus</a> sur nos fonctionnalités, ou <a href="">Téléchargez Firefox</a> dès maintenant.</p>
-
+<div class="hide-from-firefox">
+  <h2>Vous n'avez pas encore Firefox ?</h2>
+  <p><a href="{{ url('firefox.features.index') }}" data-cta-link="Learn more about our features">En savoir plus</a> sur nos fonctionnalités, ou <a href="{{ url('firefox.new') }}" data-link-text="Download Firefox" data-link-position="content">Téléchargez Firefox</a> dès maintenant.</p>
```

Comment:
**Question:** Does the first anchor tag in this line also need a `data-link-position`?